qt/Application.cc

```diff
@@ -81,13 +81,13 @@ bool loadTranslation(QTranslator& translator, QString const& name, QLocale const
 [[nodiscard]] auto makeWindowIcon()
 {
     // first, try to load it from the system theme
-    if (auto icon = QIcon::fromTheme(QStringLiteral("transmission")); !icon.isNull())
+    if (auto icon = QIcon::fromTheme(QStringLiteral(":/icons/transmission-tray.ico")); !icon.isNull())
```

Comment:
I don't know how it works in the first place (as I don't see this behavior documented), but passing a resource file path instead of a name to `QIcon::fromTheme()` is missing the point as it'll be impossible to provide a different icon via a theme. Was that your intention? (Appears so from the discussion in #4420.) If not, I see that `QIcon::themeSearchPaths()` is documented to always have `:/icons` as a fallback on all platforms, so using just the name should probably work too:
```suggestion
    if (auto icon = QIcon::fromTheme(QStringLiteral("transmission-tray")); !icon.isNull())
```

OTOH I'm not convinced that "transmission-tray" is a good choice for the window icon. While I'm kind of okay with tray icon being monochrome (as long as it's recognizable enough), I think window icons are usually full-color. Could you provide more context as to why this change is necessary?

Reply:
Many thanks for taking the time to look at this Mike. Yes, overall, i'm trying to tread a fine line here, where the tray icon should match the application icon for anyone who is building from source, and hopefully providing a path for packagers to match the app icon with the tray icon.

Re line 84, i agree with you, but that proposed change throws the tray icon back to whatever theme is in play, and i don't know how to uncouple the tray from the window.

Reply:
Looking in AboutDialog.ui doesn't suggest a means of specifying a particular icon. This is where i'm out of my depth, where complex code changes are required to specify what icon should be in place there.